src/templates/gwfcore/gwfcore-root.template.yaml

```diff
@@ -213,6 +251,22 @@ Resources:
       Value: !Sub ${S3Stack.Outputs.BucketName} #required
       Description: GWFCore S3 Bucket
   
+  ParamEfsFileSystem:
+    Type: "AWS::SSM::Parameter"
+    Condition: EfsCondition
```

Comment:
related to the point above, could this be a guaranteed SSM parameter? Effectively, remove the conditional here. In the case of no EFS, just store an empty value (`""`) and have the provisioning script handle it accordingly.

Reply:
I think we can do it that way. Just will then make the `LaunchTemplate` stack depend on the creation of this parameter